Formula/hbase.rb

```diff
@@ -103,52 +103,18 @@ def post_install
     (var/"run/hbase").mkpath
   end
 
-  plist_options manual: "#{HOMEBREW_PREFIX}/opt/hbase/bin/start-hbase.sh"
-
-  def plist
-    <<~EOS
-      <?xml version="1.0" encoding="UTF-8"?>
-      <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
-      <plist version="1.0">
-      <dict>
-        <key>KeepAlive</key>
-        <true/>
-        <key>Label</key>
-        <string>#{plist_name}</string>
-        <key>EnvironmentVariables</key>
-        <dict>
-         <key>HBASE_MASTER_OPTS</key><string> -XX:PermSize=128m -XX:MaxPermSize=128m</string>
-         <key>HBASE_LOG_DIR</key><string>#{var}/hbase</string>
-         <key>HBASE_HOME</key><string>#{opt_libexec}</string>
-         <key>HBASE_SECURITY_LOGGER</key><string>INFO,RFAS</string>
-         <key>HBASE_PID_DIR</key><string>#{var}/run/hbase</string>
-         <key>HBASE_NICENESS</key><string>0</string>
-         <key>HBASE_IDENT_STRING</key><string>root</string>
-         <key>HBASE_REGIONSERVER_OPTS</key><string> -XX:PermSize=128m -XX:MaxPermSize=128m</string>
-         <key>HBASE_OPTS</key><string>-XX:+UseConcMarkSweepGC</string>
-         <key>HBASE_ROOT_LOGGER</key><string>INFO,RFA</string>
-         <key>HBASE_LOG_PREFIX</key><string>hbase-root-master</string>
-         <key>HBASE_LOGFILE</key><string>hbase-root-master.log</string>
-        </dict>
-        <key>ProgramArguments</key>
-        <array>
-          <string>#{opt_bin}/hbase</string>
-          <string>--config</string>
-          <string>#{opt_libexec}/conf</string>
-          <string>master</string>
-          <string>start</string>
-        </array>
-        <key>RunAtLoad</key>
-        <true/>
-        <key>WorkingDirectory</key>
-        <string>#{HOMEBREW_PREFIX}</string>
-        <key>StandardOutPath</key>
-        <string>#{var}/hbase/hbase.log</string>
-        <key>StandardErrorPath</key>
-        <string>#{var}/hbase/hbase.err</string>
-      </dict>
-      </plist>
-    EOS
+  service do
+    run [opt_bin/"hbase", "--config", opt_libexec/"conf", "master", "start"]
+    keep_alive true
+    working_dir HOMEBREW_PREFIX
+    log_path var/"hbase/hbase.log"
+    error_log_path var/"hbase/hbase.err"
+    environment_variables HBASE_MASTER_OPTS: " -XX:PermSize=128m -XX:MaxPermSize=128m", HBASE_LOG_DIR: var/"hbase",
+                          HBASE_HOME: opt_libexec, HBASE_SECURITY_LOGGER: "INFO,RFAS", HBASE_PID_DIR: var/"run/hbase",
+                          HBASE_NICENESS: "0", HBASE_IDENT_STRING: "root",
+                          HBASE_REGIONSERVER_OPTS: " -XX:PermSize=128m -XX:MaxPermSize=128m",
+                          HBASE_OPTS: "-XX:+UseConcMarkSweepGC", HBASE_ROOT_LOGGER: "INFO,RFA",
+                          HBASE_LOG_PREFIX: "hbase-root-master", HBASE_LOGFILE: "hbase-root-master.log"
```

Comment:
Can you put all arguments on one line?

Reply:
collapse this all down to a single line? that fails the audit: `Line is too long`

or do you mean each key/value pair on a separate line?

Reply:
I meant each their own line.